### PR TITLE
TN-4317: Add import used versions of packages feature.

### PR DESCRIPTION
**Pull request by fisgeci**
#### Description

Please include a description of the change and why the change was made.

#### Checklist

- [x] I have self-reviewed this PR
- [x] I have tested the change and proved that it works in different scenarios
- [x] I have updated docs if needed

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -276,6 +276,34 @@ You can use the --includeDependencies flag to also export the dependencies of th
...
+You can use the `import packages` command to import multiple packages that were exported using `export packages` at once into studio.
+
+```
+//Importing multiple packages at once
+content-cli import packages -p <your-chosen-profile> --packagesFile <path to exported zip file>
+```
+
+Example usage would be:
+```
+content-cli import packages -p <your-chosen-profile> --packagesFile export_123123-13123-123123.zip
```
Comment (promeris) [inline DOCUMENTATION.md:290]: Can we simplify the file name to <exportedPackagesFile>?
```suggestion
content-cli import packages -p <your-chosen-profile> --packagesFile <exportedPackagesFile>
```

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -276,6 +276,34 @@ You can use the --includeDependencies flag to also export the dependencies of th
 content-cli export packages -p <your-chosen-profile> --packageKeys <package1> <package2> --includeDependencies
 ```
 
+### Importing multiple packages into Studio
+
+You can use the `import packages` command to import multiple packages that were exported using `export packages` at once into studio.
+
+```
+//Importing multiple packages at once
+content-cli import packages -p <your-chosen-profile> --packagesFile <path to exported zip file>
```
Comment (promeris) [inline DOCUMENTATION.md:285]:
```suggestion
content-cli import packages -p <your-chosen-profile> --packagesFile <exportedPackagesFile>
```

File: DOCUMENTATION.md [hunk 3]
```diff
@@ -276,6 +276,27 @@ You can use the --includeDependencies flag to also export the dependencies of th
...
+```
+
+Example usage would be:
+```
+content-cli import packages -p <your-chosen-profile> --packagesFile export_123123-13123-123123.zip
+```
+
+You can also use the `--spaceMapping` flag to provide a mapping of packages to spaces in target team:
```
Comment (promeris) [inline DOCUMENTATION.md:293]:
```suggestion
You can also use the `--spaceMappings` flag to provide a mapping of packages to spaces in target team:
```

File: DOCUMENTATION.md [hunk 4]
```diff
@@ -276,6 +276,27 @@ You can use the --includeDependencies flag to also export the dependencies of th
...
+
+```
+// Example usage of spaceMapping
+content-cli import packages -p <your-chosen-profile> --packagesFile <path to exported zip file> --spaceMapping <package-key-1>:<target-space-key-1> <package-key-2>:<target-space-key-2> ...
```
Comment (promeris) [inline DOCUMENTATION.md:297]:
```suggestion
content-cli import packages -p <your-chosen-profile> --packagesFile <exportedPackagesFile> --spaceMappings <package-key-1>:<target-space-key-1> <package-key-2>:<target-space-key-2> ...
```

File: src/services/file-service.ts [hunk 5]
```diff
@@ -2,6 +2,7 @@ import * as fs from "fs";
 import * as path from "path";
 import * as YAML from "yaml";
 import {ManifestNodeTransport} from "../interfaces/manifest-transport";
+import {Stream} from "stream";
```
Comment (promeris) [inline src/services/file-service.ts:5]: unused import?

**Review by promeris — DISMISSED** [review]

**Review by promeris — DISMISSED** [review]

**Review by AlbinHoxha — APPROVED** [review]

**Pull request merged**